ui/src/textbox.tsx

```diff
@@ -17,6 +17,14 @@ import { B, Id, S } from './core'
 import React from 'react'
 import { debounce, wave } from './ui'
 
+const hideFieldArrowsStyle = {
```

Comment:
Keep the native browser behavior - https://github.com/h2oai/wave/issues/2170#issuecomment-1781226049